recipes/scg/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "scg" %}
+{% set version = "0.3.1" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://bitbucket.org/aroth85/scg/downloads/{{ name }}-{{ version }}.tar.gz
+    md5: 6f0fb91266de66d6a9d73fb741f49e13
+
+build:
+    script: $PYTHON setup.py install
```

Comment:
can you add `--single-version-externally-managed --record=record.txt` to the end of setup.py